### PR TITLE
[New paper] Add a whitepaper on Neural Network Quantization

### PR DESCRIPTION
**Pull request by p-wysocki**
name: A White Paper on Neural Network Quantization
about: Quantization - from absolute basics to advanced concepts
labels: quantization, miscellaneous

**Topic:** *quantization*

**Paper title:** *A White Paper on Neural Network Quantization*

**Year:** *2021*

**Author(s):** *Nagel et al.*

**Canonical link:** *https://arxiv.org/abs/2106.08295*

**Notes:** It's a fantastic paper for geting into neural network quantization. It covers everything from absolute basics (including quantization justification explained in terms of hardware) up to advanced quantization techniques. I found it exceptionally useful and clear.

### CODE REVIEW

**Comment by daturkel** [comment]
This looks like a great paper, though I typically try to link to the paper that originated or largely popularized a concept. In the case of quantization, that might be [BinaryConnect](https://arxiv.org/abs/1511.00363)? If that seems like an appropriate starting point, I could include both BinaryConnect and the Qualcomm white paper, which also seems to be one of the heavily-cited papers on the topic.

**Comment by p-wysocki** [comment]
I'm happy with whatever you decide. BinaryConnect seems interesting, I'll read it sometime soon, and also include in this PR if you want. I get that this paper may not fit too well in this repository, I put it because I believe it's a [major popularization of a technique](https://github.com/daturkel/learning-papers/blob/master/README.md?plain=1#L15) - it's basically an ELI5 of quantization :) Please let me know what you decide.
